python/paddle/fluid/executor.py

```diff
@@ -1555,23 +1555,32 @@ def _can_use_interpreter_core(program, place):
                     place, core.CustomPlace):
                 return False
 
-            use_standalone_executor_for_compiled_program = os.environ.get(
+            use_standalone_executor_for_distribution = os.environ.get(
```

Comment:
NIT, why change to for distribution?

Reply:
After this PR, the FLAGS_CONVERT_GRAPH_TO_PROGRAM is no longer control whether to run Compiled Program with Standalone Executor, but control whether to run distribution. 
It is better to change the FLAGS name at the same time. However, considering that it is just a temporary FLAGS and QAs have been set it in their testing environment, keeping the same FLAGS name can avoid the requirement for QA to reset FLAGS when they try to test the distributed scenario of Standalone Executor.